pkg/operator/operator.go

```diff
@@ -336,6 +336,9 @@ func New(
 		controllerRef,
 	)
 
+	// Set event recorder
```

Comment:
@simonpasquier I did it like this since we need a k8s client to be able to instantiate an event recorder. Let me know if it's OK or if there's a better alternative.